src/puppetlabs/ezbake/dependency_utils.clj

```diff
@@ -63,8 +63,8 @@
 
 (defn expand-snapshot-version
   ([lein-project coords] (expand-snapshot-version lein-project coords {}))
-  ([lein-project [dependency version :as coords] options]
-   [dependency (get-full-snapshot-version lein-project coords options)]))
+  ([lein-project coords options]
+   (assoc coords 1 (get-full-snapshot-version lein-project coords options))))
```

Comment:
This function could probably use a docstring, since it's a little non-intuitive that it's trying to preserve :exclusions